gitpod-shared/src/features.ts

```diff
@@ -416,7 +440,9 @@ export async function registerWorkspaceTimeout(context: GitpodExtensionContext):
 			return;
 		}
 		extendTimeoutStatusBarItem.tooltip = `Workspace Timeout: ${instance.status.timeout}. Click to extend.`;
-		extendTimeoutStatusBarItem.color = instance.status.timeout === '180m' ? new vscode.ThemeColor('notificationsWarningIcon.foreground') : undefined;
+
+		// TODO: query default timeout, currently all paid plan default timeout is 60m. 
```

Comment:
> And in the feature if the user already customizes workspace timeout time, it will also disable closed timeout, do we need to tell the user?

Interesting question. We can document this, but I'm not sure there's much we can do help the user in the product? 💭